src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/NetCorePackageReferenceProject.cs

```diff
@@ -42,6 +44,8 @@ public class NetCorePackageReferenceProject : BuildIntegratedNuGetProject
 
         private readonly IProjectSystemCache _projectSystemCache;
         private readonly UnconfiguredProject _unconfiguredProject;
+        private List<(NuGetFramework, Dictionary<string, ProjectInstalledPackages>)> _installedPackages = new List<(NuGetFramework, Dictionary<string, ProjectInstalledPackages>)>();
```

Comment:
This NetCorePackageReferenceProject change comes from Martin's [PR](https://github.com/NuGet/NuGet.Client/pull/3539). He already had PR for reference type unit tests and I'm using some of his code only for reference type project unit test.